test/moonwall.config.json

```diff
@@ -955,6 +955,59 @@
                 }
             ]
         },
+        {
+            "name": "zombie_tanssi_relay_dht_bootnodes",
```

Comment:
Should we consider this feature as important to run on merge once ready @tmpolaczyk ? I think we should, but I trust you on this one